add-ons/debezium/README.md

```diff
@@ -0,0 +1,94 @@
+# Deploying Debezium on OpenShift
+
+This addon provides an easy way to install [Debezium](http://debezium.io/) on [Minishift](https://github.com/minishift/minishift).
+
+Prerequisites:
+
+ * [Installed and running](https://docs.openshift.org/latest/minishift/getting-started/installing.html) Minishift
+ * [Installed](https://docs.openshift.org/latest/minishift/command-ref/minishift_oc-env.html) OpenShift CLI
+
+## Add-on installation
+Clone this repository onto your local machine and then install the add-on via:
+```
+    $ minishift addons install <path_to_directory_containing_this_readme>
+```
+
+## Debezium start
+Deploy the Kafka broker, Kafka Connect with Debezium and MySQL connector
+
+```
+minishift addon apply -a DEBEZIUM_VERSION=0.6.1 -a DEBEZIUM_PLUGIN=mysql -a PROJECT=myproject debezium
+```
```

Comment:
Minishift allows to give default values to variables [1] in case you are fine if user can get started with some default values e.g. `# Var-Defaults: PROJECT_USER=john`

[1] https://docs.openshift.org/latest/minishift/using/addons.html#addon-dynamic-variables

Reply:
We'd like intentionally keep version and plugin as manadatory parameters.